Homework/css/styles.css

```diff
@@ -0,0 +1,95 @@
+
+body {	
+
+	background: rgb(197,222,234); /* Old browsers */
```

Comment:
@qodesmith This section seemed bloated to me at first, but I think it's necessary in that it allows people viewing from different browsers to be able to correctly see my background content. Moz/webkit are for Mozilla Firefox and Google Chrome respectively. I know you already knew that already, but I'm just telling you what I know. Is there a shorter way to do this? Or anything else you might want to add to my knowledge around this?